.github/workflows/release.yaml

```diff
@@ -31,7 +31,47 @@ jobs:
               (cd ${dir}; helm dependency build)
           done
 
+      # push an updated index.yaml to gh-pages branch
       - name: Run chart-releaser
         uses: helm/chart-releaser-action@v1
         env:
           CR_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+
+  build-and-deploy-html:
+    needs: release
+    runs-on: ubuntu-latest
+
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+
+    permissions:
+      pages: write
+      id-token: write
+
+    # Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+    # However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+    concurrency:
+      group: "pages"
+      cancel-in-progress: false
+      
+    steps:
+      - name: Checkout gh-pages branch
+        uses: actions/checkout@v4
+        with:
+          ref: gh-pages
+
+      # build index.html from new index.yaml
+      - name: Run repo-html to build chart index.html
+        run: |
+          helm plugin install https://github.com/halkeye/helm-repo-html
```

Comment:
We could pin a version here. I'm not sure that would help, since the file that controls the version just links to the latest version of other files, not a released artifact.